lumen/ai/assistant.py

```diff
@@ -33,12 +38,24 @@ def __init__(
         agents: list[Agent | Type[Agent]] | None = None,
         **params,
     ):
+        def download_messages():
+            def filter_by_reactions(messages):
+                return [
+                    message for message in messages if "favorite" in message.reactions
+                ]
+
+            messages = self.interface.serialize(filter_by=filter_by_reactions)
```

Comment:
This should use the custom serializer https://github.com/holoviz/lumen/pull/531/files#diff-4699e4bd1882a5df97783ddd201518bcff9dc8ad2974a5c88196ffbc4c6cca38R162-R169

So it only captures the YAML specs.